taskcluster/android_taskgraph/transforms/build_apk.py

```diff
@@ -46,32 +46,59 @@ def add_shippable_secrets(config, tasks):
         secrets = task["run"].setdefault("secrets", [])
         dummy_secrets = task["run"].setdefault("dummy-secrets", [])
 
-        if task.pop("include-shippable-secrets", False) and config.params["level"] == "3":
+        if (
+            task.pop("include-shippable-secrets", False)
+            and config.params["level"] == "3"
+        ):
             gradle_build_type = task["run"]["gradle-build-type"]
-            secret_index = f'project/mobile/focus-android/{gradle_build_type}'
-            secrets.extend([{
-                "key": key,
-                "name": secret_index,
-                "path": target_file,
-            } for key, target_file in (
-                ('adjust', '.adjust_token'),
-                ('sentry_dsn', '.sentry_token'),
-                ('mls', '.mls_token'),
-                ('nimbus_url', '.nimbus'),
-            )])
+            secrets.extend(
+                [
+                    {
+                        "key": key,
+                        "name": _get_secret_index(task["name"]),
+                        "path": target_file,
+                    }
+                    for key, target_file in (
+                        ("adjust", ".adjust_token"),
+                        (
+                            "firebase",
+                            "app/src/{}/res/values/firebase.xml".format(
+                                gradle_build_type
+                            ),
+                        ),
+                        ("sentry_dsn", ".sentry_token"),
+                        ("mls", ".mls_token"),
+                        ("nimbus_url", ".nimbus"),
+                        ("wallpaper_url", ".wallpaper_url"),
+                        ("pocket_consumer_key", ".pocket_consumer_key"),
```

Comment:
Note: These 2 secrets and `firebase` didn't exist in Focus. Staging uses dummy secrets so we may hit an issue that we didn't see during testing.